.github/workflows/package_app.yml

```diff
@@ -0,0 +1,52 @@
+name: Package Application
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    paths:
+      - '**/package_app.yml'
```

Comment:
@isabelle-dr mentioned it would be useful to run this workflow on every commit in a PR so that we have access to the artifact to test the validator using the package. Thoughts?

Reply:
I [asked about this](https://github.com/MobilityData/gtfs-validator/issues/1163#issuecomment-1136251359) in issue #1163 and hadn't heard anything from @isabelle-dr there.  However, if you are hearing that it'd be helpful to run this on every PR, I'm happy to remove the path restriction.